src/preamble.js

```diff
@@ -599,7 +599,11 @@ function abort(what) {
   err(what);
 
   ABORT = true;
-  EXITSTATUS = 1;
+  EXITSTATUS = 134; // 128 + 6(SIGABRT)
+
+#if expectToReceiveOnModule('onExit')
+  if (Module['onExit']) Module['onExit'](EXITSTATUS);
```

Comment:
This is an increase in default code size as by default we do receive `onExit`. That's unfortunate as we should try to have fewer of these things rather than more. But i can't find a way around that, if we want to fix this - we can't gate it on `ASSERTIONS` or anything like that.

Do you think this is strongly worth fixing in general? If the main reason is the test suite issue, I think we can fix that by adding `Module.onAbort` alongside `Module.onExit` which we already have. That would avoid a code size increase.

Reply:
Sorry I was trying (unsuccessfully again) to stack changes in github